src/client-private.ts

```diff
@@ -73,292 +56,19 @@ export async function setupStreamingClient (this: PureCloudWebrtcSdk): Promise<v
  * @param this must be called with a PureCloudWebrtcSdk as `this`
  */
 export function proxyStreamingClientEvents (this: PureCloudWebrtcSdk) {
+  this.sessionManager = new SessionManager(this);
+  this.sessionManager.initSessionHandlers();
```

Comment:
I can try that. I didn't do that initially because the sessionHandlers require a handle on then sessionManager and I wasn't sure if that would jive in the sessionManager's constructor.